Installing-and-Configuring/OS/Qubes.md

```diff
@@ -93,13 +41,17 @@ If that completes with no errors it will launch the Xen hypervisor from the
 
 Use default QubesOS partitioning scheme for QubesOS 4.x
 
-![Disk Recovery Key]({{ site.baseurl }}/images/Disk_encryption_recovery_key.jpg)
+![Disk Recovery Key]({ site.baseurl }}/images/Disk_encryption_recovery_key.jpg)
 
 The Disk Recovery Key that you enter here will be used as the
 "recovery password" later.  It should be a long value since you won't
 have to enter it very often; only when upgrading the Heads firmware
 or if there is a need to recover the disk on an external machine.
-You will need it again shortly, so don't lose it yet.
+
+DO NOT lose the Disk recovery key. This key passphrase will need to be [reentered](/Keys/#tpm-disk-encryption-key).
```

Comment:
Do not loose the Disk Recovery key -> Do not loose the Disk Recovery key

Reply:
?  you have misspelled lose and repeated the same sentence.  Please clarify.

Reply:
@tlaurion This is the only one not addressed.  Not sure what this is.

Reply:
modified https://github.com/osresearch/heads-wiki/pull/76#discussion_r702474772

Sorry, learned a thing in english today. But you can't lose the Key really. The block above says the key will be used as passphrase later... This is confusing even for me, I cannot see how it can be clear for others.

But this is true, per man cryptsetup:
"Each passphrase, also called a key in this document, is associated with
       one of 8 key-slots.  Key operations that do not specify a  slot  affect
       the  first slot that matches the supplied passphrase or the first empty
       slot if a new passphrase is added."
       
 So key == passphrase here.... 
 While my point was that we remember a passphrase, not the binary form stored in the slot which is the key. But we might not mind.